### PR TITLE
Change remaining tests so they pass

### PR DESCRIPTION
**Pull request by aaronlehmann**
Also, fix some related issues.

Remove client_test.go because it depends on external resources that are incompatible.

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -57,12 +57,12 @@ func (c *Client) Update() error {
...
 	err := c.downloadTimestamp()
 	if err != nil {
-		logrus.Errorf("Client Update (Timestamp): ", err.Error())
+		logrus.Errorf("Client Update (Timestamp): %s", err.Error())
 		return err
 	}
 	err = c.downloadSnapshot()
 	if err != nil {
-		logrus.Errorf("Client Update (Snapshot): ", err.Error())
+		logrus.Errorf("Client Update (Snapshot): %s", err.Error())
```
Comment (diogomonica) [inline client/client.go:65]: why %s? Why not %v and removing .Error()

File: testutils/utils.go [hunk 2]
```diff
@@ -23,12 +23,16 @@ func SampleMeta() data.FileMeta {
 }
 
 func GetSqliteDB() *sql.DB {
+	os.Mkdir("/tmp/sqlite", 0777)
```
Comment (diogomonica) [inline testutils/utils.go]: why 0777?!

**Comment by diogomonica** [comment]
LGTM.

File: testutils/utils.go (hunk 2 above)
Comment (aaronlehmann) [inline testutils/utils.go]: I figured we might as well rely on the umask to determine the permissions. But you're right; there's no good reason to use 0777. I'll change this.

**Pull request merged**